cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -1502,6 +1544,62 @@
       },
       "Condition": "CreateEC2IAMRole"
     },
+    "ParallelClusterHITPolicies": {
+      "Type": "AWS::IAM::Policy",
+      "Properties": {
+        "PolicyName": "parallelcluster-hit",
+        "PolicyDocument": {
+          "Statement": [
+            {
+              "Sid": "EC2Terminate",
+              "Effect": "Allow",
+              "Action": [
+                "ec2:TerminateInstances"
+              ],
+              "Resource": "*",
+              "Condition": {
+                "StringEquals": {
+                  "ec2:ResourceTag/Application": {
+                    "Ref": "AWS::StackName"
+                  }
+                }
+              }
+            },
+            {
+              "Sid": "EC2",
+              "Effect": "Allow",
+              "Action": [
+                "ec2:DescribeInstances",
+                "ec2:DescribeLaunchTemplates",
+                "ec2:RunInstances",
+                "ec2:DescribeInstanceStatus",
+                "ec2:RunInstances"
```

Comment:
Missing `ec2:CreateTags` permission here, so `run-instances` call will fail. Also this `ec2:RunInstances` permission is redundant, please replace with `ec2:CreateTags`